client/src/components/CustomFields.js

```diff
@@ -861,13 +866,28 @@ const filterDeprecatedFields = fieldsConfig => {
   return deprecatedFieldsFiltered
 }
 
+const isFieldValueNotSet = value => {
+  return (
+    value === undefined ||
+    value === null ||
```

Comment:
Maybe add `value === NaN ||` too?

Reply:
I think a value is never evaluated to NaN but I add the condition `isNaN(value)` to be on the safe side.

Reply:
But `isNaN(value)` coerces e.g. strings to numbers first; better to use `Number.isNaN(value)` as that only checks whether value is a number that is NaN.